iroha-balancer-core/src/main/java/soramitsu/irohautils/balancer/routes/IrohaBalancerRoute.java

```diff
@@ -74,25 +75,24 @@ public void configure() throws Exception {
                 .unmarshal().json(JsonLibrary.Jackson, ByteArrayList.class)
                 .process(exchange -> {
                     ByteArrayList body = exchange.getIn().getBody(ByteArrayList.class);
-                    log.debug("Receive transactions to listTorii, size: {}", body.size());
+                    log.info("Received transactions from listTorii, size: {}", body.size());
```

Comment:
...which are for...

Reply:
what do you mean? you receive a transaction from a queue for torii/listTorii and send it to Iroha's torii/listTorii
doesn't matter imo